book/src/offline-signing/README.md

```diff
@@ -0,0 +1,78 @@
+# Offline Transaction Signing
+
+Some security models require keeping signing keys, and thus the signing
+process, separated from transaction creation and network broadcast. Examples
+include:
+  * Collecting signatures from geographically disparate signers in a
+multi-signature scheme
+  * Signing transactions using an [airgapped](https://en.wikipedia.org/wiki/Air_gap_(networking))
+signing device
+
+To facilitate these operations, it is necessary to allow transaction
```

Comment:
Technically, doesn't transaction construction happen on both the networked and the air gapped machines? I think our CLI tools are making it possible to:
1. Offline sign
2. Easily send an offline signed transaction to the network

Reply:
I had this thought as well.

Reply:
Yep, "construction" should be "submission"